Pipfile.lock

```diff
@@ -93,6 +93,7 @@
         },
         "boto3": {
             "hashes": [
+                "sha256:34ee7608afa457da7bf4ad42dc53ac967b2f28769ef4ef1a0a41e1b40f35d897",
```

Comment:
Probably want to back out a pipfile.lock change that doesn't update alllll the requirements files. Also no overall hash change, so this would probably fail to deploy in OpenShift.

Reply:
@adberglund 
Removed the extra hash entry.
The file should match master now.